### PR TITLE
GUACAMOLE-728: Document SSL properties for MySQL and PostgreSQL.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: src/chapters/jdbc-auth.xml [hunk 1]
```diff
@@ -118,6 +225,135 @@
...
+                                                        matches the name of the server.</para>
+                                                </listitem>
+                                            </varlistentry>
+                                        </variablelist>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-cert-file</property></entry>
+                                    <entry>
+                                        <para>The client certificate file to use when making
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:318]: What format is this file expected to be in?

File: src/chapters/jdbc-auth.xml [hunk 2]
```diff
@@ -118,6 +225,135 @@
...
+                                            an SSL connection to the Postgres server. This
+                                            property is optional, and will be ignored if
+                                            the SSL mode is set to disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-key-file</property></entry>
+                                    <entry>
+                                        <para>The file containing the client private key
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:327]: Here, too - what format is this file expected to be in?

File: src/chapters/jdbc-auth.xml [hunk 3]
```diff
@@ -118,6 +225,135 @@
...
+                                            to be used when making an SSL-encrypted connction
+                                            to a Postgres server.  This property is optional
+                                            and will be ignore if the SSL mode is set to
+                                            disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-root-cert-file</property></entry>
+                                    <entry>
+                                        <para>The file that contains the root certificates
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:337]: And here - it's unclear from the description what type of file this property should point to.

File: src/chapters/jdbc-auth.xml [hunk 4]
```diff
@@ -118,6 +225,135 @@
...
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-key-password</property></entry>
+                                    <entry>
+                                        <para>The password that will be used to access the
+                                            client private key file. This property is
+                                            optional, and is only used if the
+                                            <property>postgresql-ssl-key-file</property>
+                                            property is set.</para>
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:351]: Is it required in all cases if `postgresql-ssl-key-file` is set?

File: src/chapters/jdbc-auth.xml [hunk 5]
```diff
@@ -118,6 +225,135 @@
...
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-root-cert-file</property></entry>
+                                    <entry>
+                                        <para>The file that contains the root certificates
+                                            against which the server certificate verification
+                                            will be performed.  This property is optional and
+                                            is only used if the SSL mode is set to 
+                                            verify-ca or verify-full.</para>
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:341]: Is this property required in all cases if `verify-ca` or `verify-full` are specified? Is the default Java truststore used?

File: src/chapters/jdbc-auth.xml [hunk 6]
```diff
@@ -98,6 +98,113 @@
...
+                                    <entry>
+                                        <para></para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>mysql-ssl-client-password</property></entry>
+                                    <entry>
+                                        <para>The password Guacamole should provide when authenticating with
+                                            the database. This is given as "some_password" in the examples
+                                            given in this chapter.</para>
```
[the remark this answers is not in the document]
Comment (necouchman) [reply src/chapters/jdbc-auth.xml]: Oops, copy-pasta...

File: src/chapters/jdbc-auth.xml (hunk 5 above)
Comment (necouchman) [reply src/chapters/jdbc-auth.xml]: Need to double-check this behavior...

Comment (necouchman) [reply src/chapters/jdbc-auth.xml]: Okay, I have tested and documented the behavior.  It is full of nuances, so I've tried to summarize it as best I can...

File: src/chapters/jdbc-auth.xml [hunk 7]
```diff
@@ -118,6 +230,140 @@
...
+                                            property is not specified and the default file
+                                            does not exist, the Postgres JDBC driver will
+                                            fail to connect to the server.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-key-password</property></entry>
+                                    <entry>
+                                        <para>The password that will be used to access the
+                                            client private key file, if the cient private
```
Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml:357]: client*

**Pull request merged**